easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per http://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'ConfigureMake'
+
+name = 'LUMPY'
+version = '0.2.13'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
+description = 'lumpy: a general probabilistic framework for structural variant discovery'
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no source_urls because the tarball has to be manually generated doing
+# git clone --recursive https://github.com/arq5x/lumpy-sv.git
```

Comment:
@pescobar These instructions are incomplete; to get exactly `0.2.13`, you need to do a `git reset --hard f194e6f` too:

```
git clone --recursive https://github.com/arq5x/lumpy-sv.git
cd lumpy-sv
git reset --hard f194e6f  # specific to v0.2.13
cd ..
mv lumpy-sv lumpy-sv-0.2.13
tar cfvz LUMPY-0.2.13.tar.gz lumpy-sv-0.2.13
```

Reply:
I have updated the instructions to create the tarball. Thanks!